README.md

```diff
@@ -1 +1,20 @@
-# awesome-books
\ No newline at end of file
+# Awesome-books
```

Comment:
- [ ] I would suggest you format your README with this Microverse [template](https://github.com/microverseinc/readme-template/blob/master/README.md) and add the missing sections from the lower parts of the template (ie. Contributing, Acknowledgement, etc.)
- [ ] For more elaborate documentation, in your README, kindly add a paragraph about the project.
- [ ] Also add instructions on how users can set up and run your app locally.